cirq/iqm/iqm_device_test.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+import cirq
+from cirq import iqm
+
+
+class TestOperationValidation:
+    adonis = iqm.Adonis()
+    q0 = cirq.GridQubit(0, 1)
+    q1 = cirq.GridQubit(1, 0)
+    q2 = cirq.GridQubit(1, 1)
+    q3 = cirq.GridQubit(1, 2)
+    q4 = cirq.GridQubit(2, 1)
+
+    def test_valid_operations(self):
+        self.adonis.validate_operation(cirq.GateOperation(cirq.X, [self.q0]))
```

Comment:
Instead of saying cirq.GateOperation(cirq.X, [self.q0]), you can just use `cirq.X(self.qo)`

Same for below.

Reply:
Done.